Source/plrmsg.cpp

```diff
@@ -25,16 +26,27 @@ namespace {
 
 uint8_t plr_msg_slot;
 _plrmsg plr_msgs[PMSG_COUNT];
+int msgs;
 
 /** Maps from player_num to text color, as used in chat messages. */
 const UiFlags TextColorFromPlayerId[MAX_PLRS + 1] = { UiFlags::ColorWhite, UiFlags::ColorWhite, UiFlags::ColorWhite, UiFlags::ColorWhite, UiFlags::ColorWhitegold };
 
+int GetChatLines(int width, char *textPtr)
+{
+	const size_t length = strlen(textPtr);
+	const string_view text { textPtr, length };

```

Comment:
conversion to string_view can usually be done implicitly.
```suggestion
int GetChatLines(int width, string_view text)
{
```